### DISCUSSION TITLE
Automate releases to an official Fedora repository

### DISCUSSION THREAD

**Discussion by gornostal** [Ulauncher/Ulauncher#882] [Packaging]
See [this](https://github.com/Ulauncher/Ulauncher/pull/536#issuecomment-612459120) thread for more info

If someone is interested in implementing this automation, let me know in the comments.

Thanks!

**Comment by recrsn** [comment]
I have some experience in packaging things in RPM and would be able to help here.

**Comment by troycurtisjr** [comment]
Fedora does have an upstream monitoring service https://fedoraproject.org/wiki/Upstream_release_monitoring which automates much of the task of pushing updated packages. It does require an approval for the final push, which it probably the most conservative approach. Do you think this addresses the automation part of this?

**Comment by troycurtisjr** [comment]
Note that I just mapped Ulauncher https://release-monitoring.org/project/141340/
